client/me/purchases/manage-purchase/style.scss

```diff
@@ -310,6 +310,18 @@
 		}
 	}
 
+	.manage-purchase__no-payment-method {
+		display: flex;
+		align-items: center;
+		gap: 4px;
+		svg {
+			flex-shrink: 0;
+			path {
+				fill: var( --color-warning-30 );
```

Comment:
Minor note: it looks like this still works (on Chrome and Firefox at least) if we push the `fill` attribute up to the `svg` element. This would make the selector slightly less dependent on the specific markup within the SVG image. (This is not blocking, but was something I wondered during this round of testing.)

Reply:
Ok, that's a bit cleaner. Done